options/locale/locale_en-US.ini

```
@@ -2312,6 +2312,8 @@ settings.protect_approvals_whitelist_users = Whitelisted reviewers:
 settings.protect_approvals_whitelist_teams = Whitelisted teams for reviews:
 settings.dismiss_stale_approvals = Dismiss stale approvals
 settings.dismiss_stale_approvals_desc = When new commits that change the content of the pull request are pushed to the branch, old approvals will be dismissed.
+settings.ignore_stale_approvals = Ignore stale approvals
+settings.ignore_stale_approvals_desc = When new commits that change the content of the pull request are pushed to the branch, old approvals will be ignored.
```

```suggestion
settings.ignore_stale_approvals_desc = Do not count approvals that were made on older commits (stale reviews) towards how many approvals the PR has.
```

Woohoo :unamused:
This seems like the perfect candidate for a three-state enum for me (`ON_NEW_COMMIT = [DO_NOTHING, IGNORE_OLD_REVIEWS, DISMISS_OLD_REVIEWS]`).
However, I can see why we need a four-state algorithm instead for backward compatibility.